stemcell_builder/stages/system_oracle/assets/etc/cloud/cloud.cfg

```diff
@@ -0,0 +1,111 @@
+# The top level settings are used as module
```

Comment:
probably should base this off of the cloud.cfg from our existing ubuntu cloud image.